docs/docs/general/02-Components/AskUI-Development-Environment.md

```diff
@@ -185,3 +186,74 @@ Note: Adjust the parameters as needed for your specific configuration.
 :::
 
 See also the dedicated [AskUI Runner docs](AskUI-Runner.md) for more information.
+
+
+## ADE + VSCode IDE
+
+The ADE is build to work seamlessly with (VSCode)[https://code.visualstudio.com/] together. Therefore we create (VScode Workspace Settings)[https://code.visualstudio.com/docs/getstarted/settings#_settingsjson]
+
+**Note: Only compatible with windows**
```

Comment:
I think this should be a fat warning so it does not go unnoticed. I think we use formatted blocks for that similar to the danger block (but in yellow to signify a warning) like in https://docs.askui.com/docs/0.13.1/general/Getting%20Started/Installing%20AskUI/getting-started-macos#requirements @JohannesDienst-askui